src/pages/patterns/index.js

```diff
@@ -15,67 +15,78 @@ import capitalize from '@freesewing/utils/capitalize'
 
 import './patterns.css'
 
-const PatternsIndexPage = (props) => {
+const PatternsIndexPage = props => {
   const app = useApp()
   useEffect(() => {
-    app.setTitle(app.translate('app.patterns'))
+    app.setTitle(app.translate('app.my_patterns'))
```

Comment:
not sure if we want this and how to add a fallback translation

Reply:
Seems like we've your **Your patterns** but not **My patterns**:

![Screenshot_2020-04-04 Patterns](https://user-images.githubusercontent.com/1708494/78421666-a9228480-7659-11ea-8d85-9249f6b5508b.png)

I seem to remember that choice was inspired by [this stackexchange answer](https://ux.stackexchange.com/a/4350) on the use of My vs Your in UI. At least in this menu, I think it would be confusing to write *You need to be logged into save my patterns*.

So the simplest fix would be to go with `yourPatterns` (note that translation terms are camelCased)

Reply:
Also, looking at the screenshot, that list of patterns could really use a delimiter between them names.

Reply:
cool on it!